core-sdk-samples/higgs-shop-sample-app/HiggsShopSampleApp/AppDelegate.swift

```diff
@@ -97,6 +98,12 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, MPListenerProtocol
                 options.proxyAppDelegate = false
             }
         }
+        
+        // Sideloaded kits are simply classes that conform to MPKitProtocol and can be used to receive callbacks when various things happen such as events being logged.
+        // This example is a simple implementation that only logs the callbacks to the console, but the data in the callbacks can be used for anything.
+        // NOTE: Sideloaded kits are always active regardless of server-side configuration.
```

Comment:
@einsteinx2 - I know you already merged, but nit - since there is not server-side configuration, i'd just say `NOTE: Sideloaded kits are always active`